samcli/commands/check/resources/Pricing.py

```diff
@@ -0,0 +1,155 @@
+"""
+Pricing questions are asked here. Pricing is only done for
+Lambda Functions as of now. Data is stored in graph in
+Lambda function pricing object.
+"""
+
+from typing import List, Tuple
+import click
+
+from samcli.commands.check.resources.Graph import Graph
+from samcli.commands.check.resources.LambdaFunctionPricing import LambdaFunctionPricing
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class Pricing:
```

Comment:
This class named as `Pricing` but it looks like it is specific to Lambda, can we rename the class name to reflect that?

Reply:
It looks like that because I only have pricing implemented for lambda functions. Eventually when pricing is implemented for more resources, the logic for them will be handled here.